build_projects/update-dependencies/Program.cs

```diff
@@ -60,6 +60,20 @@ public static void Main(string[] args)
             }
         }
 
+        private static BuildInfo GetBuildInfo(string name, string buildInfoFragment, bool fetchLatestReleaseFile = true)
+        {
+            const string FileUrlProtocol = "file://";
```

Comment:
Huh, if you're able to get a valid `file://` URI, doesn't that work with `BuildInfo.Get`?